Use the standard png-library instead of rolling our own.

	-mi

--- src/pngenc.cpp	Sun May  7 14:02:31 2006
+++ src/pngenc.cpp	Wed Jun 14 01:50:19 2006
@@ -21,10 +21,5 @@
  */
 
-#ifndef png_jmpbuf
-#  define png_jmpbuf(png_ptr) ((png_ptr)->jmpbuf)
-#endif
-
-#define ALL_STATIC
-#include <../libpng/png.h>
+#include <png.h>
 #include <stdio.h>
 #include <stdlib.h>
@@ -33,6 +28,4 @@
 #include "pngenc.h"
 #include "message.h"
-
-#undef jmpbuf
 
 static void user_error_fn(png_structp, png_const_charp error_msg)
